### ISSUE TITLE
New Request for Hurtworld game

### ISSUE THREAD

**Issue by ponasromas** [Austinb/GameQ#458]
Hello,
please add support for Hurtworld game. Thank you.

**Comment by rostov114** [comment]
Hello.

You can provide the addresses of the servers of this game?

**Comment by ponasromas** [comment]
212.24.98.1:12871
87.98.236.134:12871

**Comment by rostov114** [comment]
Created Pull request https://github.com/Austinb/GameQ/pull/460
I note the Game and Query ports are different (standart gameport = 12871, queryport = 12881).

```
Array
(
    [212.24.98.1:12881] => Array
        (
            [dedicated] => d
            [eac] => True
            [game_descr] => Hurtworld
            [game_dir] => hurtworld
            [game_id] => 393420
            [gametype] => 154
            [gq_address] => 212.24.98.1
            [gq_dedicated] => d
            [gq_gametype] => Hurtworld
            [gq_hostname] => Anon Server
            [gq_joinlink] => steam://connect/212.24.98.1:12881/
            [gq_mapname] => nullius
            [gq_maxplayers] => 10
            [gq_mod] => hurtworld
            [gq_name] => Hurtworld
            [gq_numplayers] => 0
            [gq_online] => 1
            [gq_password] => 1
            [gq_port_client] => 12881
            [gq_port_query] => 12881
            [gq_protocol] => source
            [gq_transport] => udp
            [gq_type] => hurtworld
            [hostname] => Anon Server
            [lastwipe] => 1549030893
            [map] => nullius
            [max_players] => 10
            [nextwipe] => -1
            [num_bots] => 0
            [num_players] => 0
            [num_rules] => 6
            [os] => l
            [password] => 1
            [players] => Array
                (
                )

            [port] => 12871
            [protocol] => 17
            [queuesize] => 0
            [rpmc] => 0
            [secure] => 1
            [steam_id] => 90123149630760963
            [steamappid] => 0
            [teams] => Array
                (
                )

            [version] => 154
        )

)
```

**Comment by ponasromas** [comment]
Is there any methodology how you create such updates?

**Comment by rostov114** [comment]
Game wiki, Google :)

**Comment by Austinb** [comment]
Merged in 3085b8d
